src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -236,4 +238,25 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             contentFrame.Navigate(xaml_typename<Editor::Keybindings>());
         }
     }
+
+    void MainPage::OpenJsonTapped(IInspectable const& /*sender*/, Windows::UI::Xaml::Input::TappedRoutedEventArgs const& /*args*/)
+    {
+        const CoreWindow window = CoreWindow::GetForCurrentThread();
+        const auto rAltState = window.GetKeyState(VirtualKey::RightMenu);
+        const auto lAltState = window.GetKeyState(VirtualKey::LeftMenu);
+        const bool altPressed = WI_IsFlagSet(lAltState, CoreVirtualKeyStates::Down) ||
+                                WI_IsFlagSet(rAltState, CoreVirtualKeyStates::Down);
+
+        const auto target = altPressed ? SettingsTarget::DefaultsFile : SettingsTarget::SettingsFile;
+        _OpenJsonHandlers(nullptr, target);
+    }
+
+    void MainPage::OpenJsonKeyDown(IInspectable const& /*sender*/, Windows::UI::Xaml::Input::KeyRoutedEventArgs const& args)
+    {
+        if (args.Key() == VirtualKey::Enter)

```

Comment:
nit: do we need space here too?

Reply:
hmm i don't see a space here? also runformat didn't seem to change anything 🤷

Reply:
oh no, I mean `Spacebar` hahaha

Reply:
i think he means, "should Space activate the Open Json button"

Reply:
LOL yes, yes it should